tsdb/engine/tsm1/engine.go

```diff
@@ -422,7 +423,7 @@ func (e *DevEngine) compactTSM() {
 			}
 
 			start := time.Now()
-			e.logger.Printf("compacting %d TSM files", len(tsmFiles))
+			e.logger.Printf("beginning compaction of %d TSM files", len(tsmFiles))
```

Comment:
I don't think it's that noisy. I deliberately left it without a flag, though I don't feel super-strong about it. I would like to add logging that dumps the filenames to the logs, and *that* would be wrapped in the flag.

Make sense?